index.html

```diff
@@ -3,25 +3,26 @@
 description: Victoria BC's programmers whisky society. Connecting technical people through whisky to share their passions and pick up new technologies.
 ---
 
-<h4>Our latest Announcement:</h4>
+<h3 style="text-align: center;"><span style="text-size: +2">{</span> Who Is Whisky Oriented Development? <span style="text-size: +2">}</span></h3>
+<p>
+  We're people who work in the tech industry in Victoria, Canada.  We organize periodic whisky tasting events that seat 30 people.  In each tasting a mix of newcomers and regulars get a chance to sample some fine whisky, learn a little bit about it, and socialize.  We're also a <a href="https://whiskydev.slack.com/messages/general/">Slack</a> channel where people hang out everyday.
```

Comment:
Of note here, anyone that's not already a member of the Slack network will only get a login screen. 
<img width="1280" alt="screen shot 2016-07-21 at 7 16 22 am" src="https://cloud.githubusercontent.com/assets/46070/17026118/d4982262-4f13-11e6-803e-7093ade18acd.png">

We also have http://slack-invite.whiskydev.com/, which will allow any user to enter their email address, and be sent an invite to the slack. Unfortunately, there's no place that does both, because Slack's business mode